File: hackerrank/day01.cpp
#include <iostream>
#include <iomanip>
#include <limits>

using namespace std;

int main() {
    int i = 4;
    double d = 4.0;
    string s = "HackerRank ";
    
    // Declare second integer, double, and String variables.
    int numInteger;
    double numDouble;
    string input;
    // Read and save an integer, double, and String to your variables.
    // Note: If you have trouble reading the entire string, please go back and review the Tutorial closely.
    cin>>numInteger;
    cin>>numDouble;

    cin.ignore();
    getline(cin, input);
    // Print the sum of both integer variables on a new line.
    cout<<i+numInteger<<"\n";
    // Print the sum of the double variables on a new line.
    printf("%.1lf\n", d+numDouble);
    // Concatenate and print the String variables on a new line
    // The 's' variable above should be printed first.
    cout<<s+input<<"\n";
    
    return 0;
}

File: hackerrank/day01.swift
var i = 4
var d = 4.0
var s = "HackerRank "
// Declare second integer, double, and String variables.

// Read and save an integer, double, and String to your variables.
let inputInteger
let inputDouble
let inputString

inputInteger = readline()
inputDouble = readline()
inputString = readline()

if let a : Int = inputInteger {
    i += a;
}
if let b : Double = inputDouble {
    d += b;
}
if let c : String = inputString {
    s += c;
}
// Print the sum of both integer variables on a new line.
print(i)
// Print the sum of the double variables on a new line.
print(d)
// Concatenate and print the String variables on a new line
// The 's' variable above should be printed first.
print(s)